server/models.py

```diff
@@ -155,6 +155,10 @@ class Election(BaseModel):
     )
     standardized_contests = Column(JSON)
 
+    # When a user deletes an audit, we keep it in the database just in case
+    # they change their mind, but flag it so that we can restrict access
+    is_deleted = Column(Boolean)
```

Comment:
suggestion: either make this not-null default false, or make it `deleted_at` with a nullable timestamp, which then gives you extra information about when the delete happened.

Reply:
good call. I originally made it not-null default false, but then got lazy when working on the migration. I actually like the timestamp better - it matches the pattern we've been using - can't believe I didn't think of that!